src/main/java/athena/Ui.java

```diff
@@ -105,6 +105,15 @@ public void printTaskDeleted(Task task, String taskRestore) {
                 + taskRestore + "\nYou can thank me later\n");
     }
 
+    /**
+     * Prints out the task details that the user specified to view.
+     *
+     * @param task The task that the user wants to view.
+     */
+    public void printTaskDetails(String task) {
+        System.out.println("Here are the details of your task: " + task);
```

Comment:
How about the name `taskDetails` instead of `task`? Because we always use `task` for `Task` objects.